server/clip_server/model/clip_trt.py

```diff
@@ -34,77 +35,77 @@ def __init__(
         name: str = None,
     ):
         if name in _MODELS:
+            self._name = name
             cache_dir = os.path.expanduser(f'~/.cache/clip/{name.replace("/", "-")}')
-            self._textual_path = _download(_S3_BUCKET + _MODELS[name][0], cache_dir)
-            self._visual_path = _download(_S3_BUCKET + _MODELS[name][1], cache_dir)
+
+            self._textual_path = os.path.join(
+                cache_dir,
+                f'textual.{ONNX_MODELS[name][0][1]}.trt',
+            )
+            self._visual_path = os.path.join(
+                cache_dir,
+                f'visual.{ONNX_MODELS[name][1][1]}.trt',
+            )
+
+            if not os.path.exists(self._textual_path) or not os.path.exists(
```

Comment:
We can separate the converting into two branches: virtual and textual model converting seperatedly.

Reply:
i think they should either succeed or fail simultaneously

Reply:
The motivation is to speed up the converting when the converting is stopped by `CTRL+C`. But I'm fine to keep the current implementation which is much safe.